src/Cache/Collection.php

```diff
@@ -422,23 +399,7 @@ public function save_query_mapping_cb(
 	) {
 		$request_key = $this->build_key( $query_id, $query, $variables, $operation );
 
-		// Only store mappings of urls when it's a GET request
-		$map_the_url = false;
-		if ( isset( $_SERVER['REQUEST_METHOD'] ) && 'GET' === $_SERVER['REQUEST_METHOD'] ) {
-			$map_the_url = true;
-		}
-
-		// We don't want POSTs during mutations or nothing on the url. cause it'll purge /graphql*
-		if ( $map_the_url && ! empty( $_SERVER['REQUEST_URI'] ) ) {
-			//phpcs:ignore
-			$url_to_save = wp_unslash( $_SERVER['REQUEST_URI'] );
-
-			// Save the url this query request came in on, so we can purge it later when something changes
-			$urls = $this->store_content( $this->url_key( $request_key ), $url_to_save );
-
-			//phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log, WordPress.PHP.DevelopmentFunctions.error_log_print_r
-			error_log( "Graphql Save Urls: $request_key " . print_r( $urls, 1 ) );
-		}
+		do_action( 'wpgraphql_cache_save_request', $request_key );
```

Comment:
I feel like this action might benefit from passing more params, such as `$query_id`, `$query`, `$variables` and `$operation`.

Reply:
Would the `$this->runtime_nodes` and `$this->list_types` be useful in the action from this Collection object?

Reply:
Ya, most likely. I could see decisions needing to be made based of that data.